bot/exts/evergreen/emoji_count.py

```diff
@@ -37,7 +37,10 @@ def embed_builder(emoji: dict) -> Tuple[discord.Embed, List[str]]:
         else:
             for category_name, category_emojis in emoji.items():
                 emoji_choice = random.choice(category_emojis)
-                emoji_info = f'There are **{len(category_emojis)}** emojis in the **{category_name}** category'
+                emoji_info = (
+                    f"There {'is' if len(category_emojis) == 1 else 'are'} **{len(category_emojis)}** "
+                    f"{'emoji' if len(category_emojis) == 1 else 'emojis'} in the **{category_name}** category"
+                )
```

Comment:
Personally not a fan of this oneline. Much easier to read if you split it up into an if,else

```suggestion
if len(category_emojis) > 1:
    emoji_info = f"There are **{len(category_emojis)}** emojis in the **{category_name}** category"
else: # assuming there can't be 0 at this stage.
    emoji_info = "There is **1** emoji in the **{category_name}** category"
```

Don't commit the suggestion, definitely not indentation compatible.